### PR TITLE
javascript-loops-practice | Samuel Cho

### PR DESCRIPTION
**Pull request by SamuelCho1**
Practiced the basics of JavaScript Loops.

### CODE REVIEW

**Comment by SamuelCho1** [comment]
Screen Shot 2021-01-28 at 1 26 42 PM
Screen Shot 2021-01-28 at 1 26 53 PM
Screen Shot 2021-01-28 at 1 27 03 PM

File: javascript-loops-practice/includes-seven.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/* exported includesSeven */
+function includesSeven(array) {
+  var i = 0;
+  var value = false;
+  while (i < array.length) {
+    if (array[i] === 7) {
+      value = true;
+    }
```
Comment (codingcodymiller) [inline javascript-loops-practice/includes-seven.js:8]: This will only work if the last value in the array is `7`.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Sam! One change for you, please see my note

**Comment by SamuelCho1** [comment]
Screen Shot 2021-01-29 at 10 57 30 AM
Screen Shot 2021-01-29 at 10 57 36 AM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**